sdk/python/kfp/__init__.py

```diff
@@ -16,7 +16,7 @@
 # https://packaging.python.org/guides/packaging-namespace-packages/#pkgutil-style-namespace-packages
 __path__ = __import__("pkgutil").extend_path(__path__, __name__)
 
-__version__ = '2.0.0-alpha.4'
+__version__ = '2.0.0-alpha.5'
```

Comment:
Update the release note with the new version? We can delete 2.0.0-alpha.4 in the release note since it never work.